src/content/docs/apm/agents/net-agent/getting-started/introduction-new-relic-net.mdx

```diff
@@ -82,7 +82,7 @@ When you are ready to install, use our launcher, or see the install instructions
   To stay up-to-date with new features and bug fixes, see the [.NET agent release notes](/docs/release-notes/agent-release-notes/net-release-notes).
 </Callout>
 
-After you install the agent and wait a few minutes for your app to generate traffic, data will appear in the [APM **Summary** page](/docs/apm/applications-menu/monitoring/apm-overview-page). If [no data appears](/docs/agents/net-agent/troubleshooting/no-data-appears-net), or if you encounter other problems, see New Relic's [.NET agent troubleshooting procedures](/docs/agents/net-agent/troubleshooting).
+After you install the agent and wait a few minutes for your app to generate traffic, data will appear in the [APM **Summary** page](/docs/apm/applications-menu/monitoring/apm-overview-page). If you have problems, see [No data appears](/docs/agents/net-agent/troubleshooting/no-data-appears-net) and the other troubleshooting docs in that section. 
```

Comment:
I think you could use "...and other troubleshooting docs..." instead (drop the "the"), but it's only a slight preference.